src/Metal/MTLEnums.cs

```diff
@@ -598,15 +631,31 @@ public enum MTLFeatureSet : nuint {
 		iOS_GPUFamily2_v3 = 6,
 		[iOS (10,0), NoTV, NoWatch, NoMac]
 		iOS_GPUFamily3_v2 = 7,
+		[iOS (11,0), NoTV, NoWatch, NoMac]
+		iOS_GPUFamily1_v4 = 8,
+		[iOS (11,0), NoTV, NoWatch, NoMac]
+		iOS_GPUFamily2_v4 = 9,
+		[iOS (11,0), NoTV, NoWatch, NoMac]
+		iOS_GPUFamily3_v3 = 10,
 
-		[Mac (10,11)]
-		OSX_GPUFamily1_v1 = 10000,
+		[Mac (10,13), NoiOS, NoTV, NoWatch]
+		macOS_GPUFamily1_v1 = 10000,
```

Comment:
That's a special case, the value `10000` already exists since 10.11. IOW it's possible to write code using the new (or old) constant that will work across those versions.

In this case you can use the original `[Mac(10,11)]` availability attribute.